Subject: find-debuginfo.c: initialize fname

The newest version of elfutils has fname initialized to NULL.
We should also do this to avoid compilation error.

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 libdwfl/find-debuginfo.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libdwfl/find-debuginfo.c b/libdwfl/find-debuginfo.c
index a01293e..196271a 100644
--- a/libdwfl/find-debuginfo.c
+++ b/libdwfl/find-debuginfo.c
@@ -192,7 +192,7 @@ find_debuginfo_in_path (Dwfl_Module *mod, const char *file_name,
 	  break;
 	}
 
-      char *fname;
+      char *fname = NULL;
       int fd = try_open (dir, subdir, debuglink_file, &fname);
       if (fd < 0)
 	switch (errno)
-- 
1.7.9.5

